packages/editor/src/lib/editor/Editor.ts

```diff
@@ -1472,89 +1475,7 @@ export class Editor extends EventEmitter<TLEventMap> {
 
 	// Menus
 
-	/**
-	 * A set of strings representing any open menus. When menus are open,
-	 * certain interactions will behave differently; for example, when a
-	 * draw tool is selected and a menu is open, a pointer-down will not
-	 * create a dot (because the user is probably trying to close the menu)
-	 * however a pointer-down event followed by a drag will begin drawing
-	 * a line (because the user is BOTH trying to close the menu AND start
-	 * drawing a line).
-	 *
-	 * @public
-	 */
-	@computed getOpenMenus(): string[] {
```

Comment:
We could make this a non breaking change by callin into `this.menus.getOpenMenus` from here? Similar with other similar changes.

Reply:
That's true, and deprecate?